edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent/Uploader/FileWriter.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Writes to a local file instead of uploading.
+    /// </summary>
+    public class FileWriter : IMetricsUpload
+    {
+        public async Task UploadAsync(IEnumerable<Metric> metrics, CancellationToken cancellationToken)
+        {
+            using (FileStream file = File.Open(@"/shared/MetricsUpload.txt", FileMode.Append))
```

Comment:
why hard-coded the path?

Reply:
This is mostly here for testing purposes. For the initial offering it won't be available to the customer at all.

Reply:
Should it be configurable?

Reply:
I think its fine unless we decide to make it available to customers. It shouldn't ever ship with this option, this is just until damon's changes get into hub.